### ISSUE TITLE
building.md missing instructions how to move from compile tree to /usr tree

### ISSUE THREAD

**Issue by marksolaris** [Eloston/ungoogled-chromium#634]
### Description

The building.md docs for building finish before describing what to do with the raw build tree once completed. They need to describe or link to instructions on how to merge the new chromium into the system to replace the original chromuim. The docs are very sparse on what you actually do with the new binaries and libs. Otherwise the first time builders are marooned and wondering what to do.

### Console output and logs
```
./get_package.py ubuntu_bionic build/src/debian
cd build/src/
./debian/rules setup-local-src
dpkg-buildpackage -b -uc
```
What now?

The instructions should finish up the guidance and discuss what to do with the debs that were created:
(This recursive path in the prompt is real, I don't know why the build made that tree)
```
ubuntu:/local/ungoogle-chrominum/ungoogled-chromium-master/build/src/ungoogled-chromium-master/build/src/ungoogled-chromium-master/build/src/ungoogled-chromium-master/build/src root# cd ..; ls -alF
total 77092
drwxr-xr-x  4 root root     4096 Dec 17 21:56 ./
drwxr-xr-x 10 root root     4096 Dec 17 15:41 ../
drwxr-xr-x  2 root root     4096 Dec 17 15:42 download_cache/
drwxr-xr-x 59 root root     4096 Dec 17 21:56 src/
-rw-r--r--  1 root root   241432 Dec 17 21:55 ungoogled-chromium-common_71.0.3578.98-3\~bionic_amd64.deb
-rw-r--r--  1 root root  2312192 Dec 17 21:55 ungoogled-chromium-driver_71.0.3578.98-3\~bionic_amd64.deb
-rw-r--r--  1 root root  2864532 Dec 17 21:56 ungoogled-chromium-l10n_71.0.3578.98-3\~bionic_all.deb
-rw-r--r--  1 root root    81964 Dec 17 21:55 ungoogled-chromium-sandbox_71.0.3578.98-3\~bionic_amd64.deb
-rw-r--r--  1 root root 29213100 Dec 17 21:56 ungoogled-chromium-shell_71.0.3578.98-3\~bionic_amd64.deb
-rw-r--r--  1 root root    21607 Dec 17 21:56 ungoogled-chromium_71.0.3578.98-3\~bionic_amd64.buildinfo
-rw-r--r--  1 root root     3441 Dec 17 21:56 ungoogled-chromium_71.0.3578.98-3\~bionic_amd64.changes
-rw-r--r--  1 root root 44170112 Dec 17 21:56 ungoogled-chromium_71.0.3578.98-3\~bionic_amd64.deb

ubuntu:/local/ungoogle-chrominum/ungoogled-chromium-master/build/src/ungoogled-chromium-master/build/src/ungoogled-chromium-master/build/src/ungoogled-chromium-master/build root# dpkg -i ungoogled-chromium-common* ungoogled-chromium-driver* ungoogled-chromium-l10n* ungoogled-chromium-sandbox* ungoogled-chromium-shell* ungoogled-chromium_71.0.3578.98-3~bionic_amd64.deb
```
### Environment details

* ungoogled-chromium version: 71.0.3578.98-2
* Configuration bundle: debian_bionic for ubuntu 18.04
* Packaging type (if applicable): ?
* OS and version for building: Ubuntu 18.04
* Compiler toolchain: llvm 8

*(If relevant, add other environment details here)*

### Building and packaging steps

cd /local/ungoogle-chrome
rm -rf ungoogled-chromium-master
unzip /var/tmp/ungoogled-chromium-master_2018_12_17.zip
cd ungoogled-chromium-master/
mkdir -p build/src
./get_package.py ubuntu_bionic build/src/debian
cd build/src/
./debian/rules setup-local-src
setenv PATH "/root/bin:/usr/local/bin:/bin:/etc:/usr/etc:/usr/bin:/usr/bin/X11:/usr/local/sbin:/sbin:/usr/local/go/bin:/root/go/bin:/usr/local/etc:/usr/sbin"
dpkg-buildpackage -b -uc

**Comment by Eloston** [comment]
Good point. These instructions should also be available to all users who use Debian packages. However, I don't know the best place to put these instructions; Any suggestions?

**Comment by marksolaris** [comment]
Mostly I'm trying to make the experience less obtuse, so getting from wanting the software functionality to being able to install and run it is less of a google-fu test and more of a HOWTO recipe. I've recently switched from Redhat systems to Debian systems so have a new build eco-system to learn. Before that I've compiled 1000's of packages in other OS's and am quite fond of the old faithful `make install`. If that's not available in ungoogled-chromium, then at least having the end to end process documented will greatly help the softwares uptake.
